src/Compute/Compute/Generated/DiskEncryptionSet/Config/NewAzureRmDiskEncryptionSetConfigCommand.cs

```diff
@@ -73,6 +73,12 @@ public partial class NewAzureRmDiskEncryptionSetConfigCommand : Microsoft.Azure.
         [PSArgumentCompleter("EncryptionAtRestWithPlatformAndCustomerKeys", "EncryptionAtRestWithCustomerKey")]
         public string EncryptionType { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true)]
+        public bool? RotationToLatestKeyVersionEnabled { get; set; }
```

Comment:
Please add help message. Besides, the type of RotationToLatestKeyVersionEnabled need discuss.

Reply:
Added help message. RotationToLatestKeyVersionEnabled is mentioned as bool? in design doc.